src/Docs/Resources/current/2-internals/1-core/20-data-abstraction-layer/020-search.md

```diff
@@ -37,6 +37,14 @@ $criteria->addFilter(
 );
 ```
 
+### Adding associations to your criteria
```

Comment:
I would see a h2 `Associations` rather than putting this under the `Fields` section.